test/functional/tests/failover_standby/test_failover_advanced.py

```diff
@@ -0,0 +1,205 @@
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import pytest
+
+from api.cas.cache_config import CacheMode, CleaningPolicy, CacheModeTrait, CacheLineSize
+
+
+@pytest.mark.skip(reason="not implemented")
+@pytest.mark.multidut(2)
+@pytest.mark.parametrize("cache_mode", CacheMode.with_traits(CacheModeTrait.LazyWrites))
+@pytest.mark.parametrize("cls", [CacheLineSize.LINE_4KiB, CacheLineSize.LINE_64KiB])
+@pytest.mark.parametrize("cleaning_policy", CleaningPolicy)
+def test_failover_during_background_cleaning(pyocf_ctx, cache_mode, cls, cleaning_policy):
+    """
+    title: Failover sequence with background cleaning:
+    description:
+      Verify proper failover behaviour and data integrity after power failure during background
+      cleaning running.
+    pass_criteria:
+      - Failover procedure success
+      - Data integrity is maintained
+    parametrizations:
+      - cache mode: all cache modes with lazy writes - to make sure dirty data is produced so that
+        metadata synchronization between hosts occurs
+      - cacheline size: 4K, 64K - to test both sector I/O and full-cacheline I/O
+      - cleaning policy - as different policies have separate metadata handling implementation
+    steps:
+      - On 2 DUTs (main and backup) prepare RAID1 cache devices of 1GiB size, comprising of 2
+        Optane drives each.
+      - On 2 DUTs (main and backup) prepare primary storage device of size 1.5GiB
+      - On main DUT prefill primary storage device with random data
+      - Start a standby cache instance on the backup DUT with parametrized cacheline size
+      - Configure DRBD to replicate cache and core storage from main to backup node
+      - On main DUT:
+          - Start cache on top of cache DRBD device with parametrized cacheline size and cache mode
+          - Set cleaning policy set to NOP
+          - Wait for DRBD synchronization
+          - Fill cache with random 50% read/write mix workload, block size 4K
+          - Verify cache is > 25% dirty
+          - Switch to WO cache mode without flush
+          - Calculate checksum of CAS exported object
+          - Switch back to the parametrized cache mode without flush
+          - Switch to parametrized cleaning policy
+          - Wait for the background cleaner to start working (no wait for ACP, according to
+            policy parameters for ALRU)
+          - Verify cleaner is progressing by inspecting dirty statistics
+          - Cut power to the main DUT
+      - On backup DUT:
```

Comment:
after main dut is back (power on) ?

Reply:
no, while it's offline - I will explicitly specify the main dut power on /off point